.ycm_extra_conf.py

```diff
@@ -221,7 +221,7 @@ def Settings(**kwargs):
     and expects a dict of language-specific settings.
     """
     if kwargs['language'] == 'cfamily':
-        cfamily_settings(kwargs['filename'], **kwargs)
+        cfamily_settings(**kwargs)
```

Comment:
My python knowledge is not spectacular but I think this is the proper fix?
FYI I'm not sure if the comment at the top start of `cfamily_settings` (line 154) should be updated?